### ISSUE TITLE
Text gets blurred after text editing looses focus

### ISSUE THREAD

**Issue by blipp** [excalidraw/excalidraw#1514]
### How to reproduce
1. start with fresh canvas
2. enable the text tool, choose the “code” font
3. click in the empty space to insert text
4. type "hallo"
5. click somewhere else to end the text entering/loose focus from the text object

### What happens
The text gets a bit blurry.

### What should happen
The text should stay sharp

### Environment
Ubuntu 19.10, Firefox 75.0

After step 4 | After step 5
------------ | -------------
![Screenshot from 2020-04-29 20-43-12](https://user-images.githubusercontent.com/138279/80634661-165be680-8a5b-11ea-9e3c-90af1773719a.png) | ![Screenshot from 2020-04-29 20-43-23](https://user-images.githubusercontent.com/138279/80634657-152ab980-8a5b-11ea-9d1f-04b490d1749c.png)

**Comment by dwelle** [comment]
Yes, this is super annoying. It's likely caused by subpixel rendering (although we should be rounding it before render, so technically it shouldn't), and depends on where you happen to position text. Sometimes it is blurred , sometimes not.

**Comment by dwelle** [comment]
Just checked, and it's indeed caused by subpixels:

|x: 600.5px|x: 600px|
|--|--|
|![image](https://user-images.githubusercontent.com/5153846/80640759-2b894300-8a64-11ea-91cc-e13c2de1911b.png)|![image](https://user-images.githubusercontent.com/5153846/80640764-2d530680-8a64-11ea-9712-ea0ec71c995b.png)

**Comment by agrimball** [comment]
Still an issue. :(

**Comment by kpietraszko** [comment]
I'd like to note that excalidraw's closest competition, which is tldraw, has proper subpixel font rendering. Regardless of whether the text is selected. That's basically because text is in the DOM at all times.

**Comment by ArthV** [comment]
Hi, 

I'm facing the same issue. Using the latest version of Chrome on MacOS, when I try to reduce the resolution to 1280x720, the text that is not selected becomes blurry/fuzzy.

 <img width="386" alt="Image" src="https://github.com/user-attachments/assets/a6d4563d-a246-48a1-9456-9b3e28539c97" />

**Comment by dwelle** [comment]
It's unclear what we can do about this, although currently we have anti-aliasing disabled, which may cause sharp edges on text — though it should help with blurry text. Sometimes it tends to help with neither, so it's a toss :/.
